portfolio/src/main/webapp/projects.html

```diff
@@ -57,26 +57,26 @@
           <div id="projects-div">
             <h3 class="subtitle">My Projects & Roles</h3>
           
-            <h4 onClick="toggleDropDown('project-dropdown-1')" class="dropdown-click">Spam Slayer Web App, Sep 2019</h4>
+            <h4 onClick="toggleDropDown('project-dropdown-1')" class="dropdown-click">Spam Slayer Web App, Sep 2019<i class="arrow down" float="right"></i></h4>
             <p id="project-dropdown-1" class="dropdown-content">I made Spam Slayer with a couple of friends at HopHacks. 
               This web application is a Amazon review filter that, given a product URL, 
               detects fake reviews using an SCNN algorithm and displays genuine reviews. 
               It is not yet deployed, but you can check out how it works at this 
             <a href="https://www.youtube.com/watch?v=TvOOvH1l8Fc">YouTube link</a>.</p>
           
-            <h4 onClick="toggleDropDown('project-dropdown-2')" class="dropdown-click">Intern at Megapro Corporation, May 2019 - Aug 2019</h4>
+            <h4 onClick="toggleDropDown('project-dropdown-2')" class="dropdown-click">Intern at Megapro Corporation, May 2019 - Aug 2019<i class="arrow down" float="right"></i></h4>
```

Comment:
Maybe add the "float" as a CSS rule to the class instead?

Reply:
Right I added it to CSS but forgot to remove it in html lol